docs/DeveloperGuide.adoc

```diff
@@ -729,8 +729,60 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |add a contact picture for my contacts |identify them more easily
+
+|`* * *` |user |categorise my contacts into groups |view my contacts by relevant groups
+
+|`* * *` |user |export and send my own contact information |share my own contact information with my friends more easily
+
+|`* * *` |user |add birthdays to my contacts |remember them easily
+
+|`* * *` |user with many contacts |filter my contacts by tags/groups etc. |find relevant contacts more easily
```

Comment:
Isn't this a specialised version of `|* * * |user |categorise my contacts into groups |view my contacts by relevant groups` mentioned above?